Add details about enabling individual API resources using --runtime-config (#32284)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates.md

* The [deprecation policy](/docs/reference/using-api/deprecation-policy/) for Kubernetes explains
  the project's approach to removing features and components.

> * Starting in Kubernetes 1.24, new beta APIs are not enabled by default.  When enabling a beta

Reviewer: I would target this PR to the branch _dev-1.24_ and then change the text to
```suggestion
* Since Kubernetes 1.24, new beta APIs are not enabled by default.  When enabling a beta
```

  feature, you will also need to enable any associated API resources.  This is congruent to alpha

>   features today.  For example, to enable a particular resource like

Reviewer: Omit the “This is congruent to alpha features today” because “today” anchors the docs in time. We prefer to write timeless documentation (see https://developers.google.com/style/timeless-documentation for Google's take on this).

>   feature, you will also need to enable any associated API resources.
>   For example, to enable a particular resource like
>   `storage.k8s.io/v1beta1/csistoragecapacities`, set `--runtime-config=storage.k8s.io/v1beta1/csistoragecapacities`.

Reviewer: `/api/apistorage.k8s.io/v1beta1/csistoragecapacities/` is not a resource. I'd call that a collection, but you're overall enabling the API not just the **list** verb.

How about:
```suggestion
  feature, you will also need to enable at least one beta version of each associated API.
  For example, to enable the v1beta1 CSIStorageCapacity API, which has the URL path
  `/apis/storage.k8s.io/v1beta1/csistoragecapacities`, you add the command line argument
  `--runtime-config=storage.k8s.io/v1beta1/csistoragecapacities` to the kube-apiserver.
```

Author: Sometimes you have to an enable a beta API, but sometimes there is no new API type or it's simply a field on a GA API which is still enabled by default.  For instance, the server-side-apply feature has no API to enable.
